test/cert.uts

```diff
@@ -319,6 +319,8 @@ assert(x_pubNum.n == 19231328316532061413420367242571475005688288081144416166988
 x_pubNum.e == 0x10001
 
 = Cert class : Checking extensions
+x.show()
```

Comment:
Are these lines neeeded?

Reply:
No, until this part crashes again... They don't hurt